### ISSUE TITLE
to_string with indent

### ISSUE THREAD

**Issue by 89z** [w-henderson/Humphrey#99]
I would like something like this:

~~~json
{
   "month": 12,
   "day": 31
}
~~~

is it possible? I found these, but they don't seem to have an option for it:

- <https://docs.rs/humphrey_json/latest/humphrey_json/fn.to_string.html>
- <https://docs.rs/humphrey_json/latest/humphrey_json/value/enum.Value.html#method.serialize>

**Comment by w-henderson** [comment]
This is a feature I've been meaning to implement for a while but I haven't got around to it yet. I'll look into adding this and I'll get back to you.

**Comment by 89z** [comment]
thank you very much! I like your commitment to small crates, so if this was added it would be great.

**Comment by w-henderson** [comment]
I've merged the update and released it as v0.2.1, you should be able to update your dependencies now! Let me know if you run into any issues, I wrote a couple of tests to make sure it's working but there's always a chance of some strange edge-case that causes a bug.

**Comment by 89z** [comment]
This seems to do the job:

~~~rust
use humphrey_json::Value;

fn main() {
   let input = r#"
   {"name": "John Doe", "age": 43}
   "#;
   let value: Value = humphrey_json::from_str(input).unwrap();
   let output = value.serialize_pretty(1);
   println!("{}", output);
}
~~~

thank you!
